src/api-reference/direct-connects/hotel-service-2/Rate-details.markdown

```diff
@@ -0,0 +1,500 @@
+              <Rates>
+                <Rate RateTimeUnit="FullDuration" EffectiveDate="2018-10-26" ExpireDate="2018-10-27">
+                  <PaymentPolicies>
+                    <GuaranteePayment>
+                      <AcceptedPayments>
+                        <AcceptedPayment>
+                          <PaymentCard CardCode="VI"/>
+                        </AcceptedPayment>
+                      </AcceptedPayments>
+                    </GuaranteePayment>
+                  </PaymentPolicies>
+                  <Total AmountAfterTax="199.00" AmountBeforeTax="149.00" CurrencyCode="EUR" DecimalPlaces="2"/>
+                  <RateDescription>
+                    <Text>Test rate description. Both before and after tax.</Text>
+                  </RateDescription>
+                  <TPA_Extensions>
+                    <RequireSeriesCode>true</RequireSeriesCode>
+                  </TPA_Extensions>
+                </Rate>
+                <Rate RateTimeUnit="FullDuration" EffectiveDate="2018-10-27" ExpireDate="2018-10-28">
+                    <Total AmountAfterTax="149.00" AmountBeforeTax="99.00" CurrencyCode="EUR" DecimalPlaces="2"/>
+                </Rate>
+              </Rates>
+            </RoomRate>
+          </RoomRates>
+          <TimeSpan End="2018-10-28" Start="2018-10-26"/>
+          <BasicPropertyInfo ChainCode="ZZ" HotelCode="419430"/>
+        </RoomStay>
+      </RoomStays>
+      <TPA_Extensions RateDetailsInd="true"></TPA_Extensions>
+    </OTA_HotelAvailRS>
+  </soap:Body>
+</soap:Envelope>
+```
+
+### <a name="res-schema"></a>Schema
+
+#### OTA_HotelAvailRS
+
+|Name|Type|Description|
+|-----------|-----------|-------------|
+|`RoomStays`|`complex`|**Required** A collection of details on the room stay including time span of this room stay, and financial information related to the room stay, including guarantee, deposit, payment, and cancellation penalties.|
+|`TPA_Extensions/RateDetailsInd`|`boolean`|Always set to `true` for `ratedetails`.|
+
+#### <a name="room-stays"></a>RoomStays
+
+|Name|Type|Description|
+|----------|-----------|-------------|
+|`RoomStay`|`complex`|**Required** Details on the room stay including time span of this room stay, and financial information related to the room stay, including guarantee, deposit, payment, and cancellation penalties. A room stay represents one (1) hotel.|
+
+#### <a name="room-stay"></a>RoomStay
+
+For a description of the relationship between the `RoomID` and `RatePlanID` refer to "Relationship between RoomID and RatePlanID".
+
+|Name|Type|Description|
+|-------------------|--------------|-------------|
+|`RoomTypes`|`complex`|**Required** Details on the room stay including guest counts, time span of this room stay, pointers to res guests, guest memberships, comments, and special requests pertaining to this particular room stay. Financial information related to the room stay, including guarantee, deposit, payment, and cancellation penalties.|
```

Comment:
Hello. Can you elaborate on what "pointers to res guests" refers to?

Reply:
I think it means "tips for the guest." It was copied from src/api-reference/direct-connects/hotel-service-2/Availability.markdown 

> Hello @henry-yang-sap . Can you elaborate on what "pointers to res guests" refers to?

I think it means "tips for the guest." It was copied from src/api-reference/direct-connects/hotel-service-2/Availability.markdown

Reply:
Perfect. Thank you.